packages/mapviewer/src/modules/menu/components/advancedTools/ImportCatalogue/ProviderList.vue

```diff
@@ -102,6 +375,42 @@ defineExpose({ goToFirst })
         .providers-list-item:hover {
             background-color: $list-item-hover-bg-color;
         }
+
+        .providers-group-header {
+            cursor: pointer;
```

Comment:
there are some bootstrap utils that can achieve that too, the more you are using bootstrap classes, the less work it will be when we switch to tailwind

So if you could do a little pass of all the rules you have below, and check if some bootstrap class would do the same, it would be nice

Reply:
now all is using bootstrap classes except `bg-body` that for some reason it doesn't work with `focus` and `hover`